src/components/RadioGroup/RadioGroup.module.scss

```diff
@@ -0,0 +1,29 @@
+.radio-group {
```

Comment:
Same comment as above, let's decide if we want to add the component-level variables here or in a separate PR.

Reply:
Updated to use form variables, and added radio specific ones too.